### ISSUE TITLE
Provide option to clear activities

### ISSUE THREAD

**Issue by arpanbag001** [a85/POSTMan-Chrome-Extension#8665]
I'm unable to clear my activities from postman workspace page.
Sometimes I might not want some activities to persist. At least owners should have the ability to clear such activities. As of now, there is no such way.

It'll be very helpful if we can clear activities like we clear history in postman workspace page.

As a workaround, I can only think of deleting account and creating a fresh one, assuming that activities will be cleared when I delete account.
However, this workaround is a really bad practice, and is only applicable for individual users, not teams.

**Comment by sankalp0o** [comment]
Hey @arpanbag001, thanks for posting here!

I'm trying to understand your requirement so we can design a better experience. What do you normally use the workspace activity for? Why do you want to clear it? Unlike history (which has actual data), activity is just a log and has no use other than to display information about what happened.

**Comment by arpanbag001** [comment]
Thanks for replying!

As for use case, my primary use case will be to remove certain audit trails, as activity includes detailed date and time information.
I don't want to have a permanent record of what I was doing at a certain time of a certain day!
It just feels odd to have any data in the internet, which is not user deletable.

Hope you understand.

**Comment by sankalp0o** [comment]
@arpanbag001 I agree you should be able to delete any user information. For now, you can delete the workspace to delete this data. If we are to allow clearing activity, we'd think about how this affects the integrity of the workspace in collaborative settings. We'd definitely want to think more about this as well as have more people weigh-in here.

**Comment by arpanbag001** [comment]
> @arpanbag001 I agree you should be able to delete any user information. For now, you can delete the workspace to delete this data. If we are to allow clearing activity, we'd think about how this affects the integrity of the workspace in collaborative settings. We'd definitely want to think more about this as well as have more people weigh-in here.

Thanks.
Just a suggestion, if you want to show users some activity is deleted, you can show a "Activity deleted", maybe even "Activity deleted by user xxxx", like WhatsApp shows for deleted messages.
Alternatively, a "Clear all activities" button is still fine by me. It also doesn't interfere with workspace integrity, as all the activities (not selective few) are deleted.
Anyway, looking forward to it.

**Comment by sagewall** [comment]
I would also like to be able to clear workspace activity.  Users are unable to delete their default "My Workspace" workspace to remove it.  I would preferably like to have the ability to completely turn off logging activity and history in the applications settings.  I work independently so having any of this logging is unnecessary and raises privacy concerns.  I actually deleted my entire Postman account just to clear out the history of "My Workspace" and created a new account.  The first thing I did with the new account was create a new private workspace that I can regularly delete to remove the activity logs.

**Comment by akshay-kulkarni-03** [comment]
@sagewall the purpose of workspace activity in to provide information to other users on what things have changed in the workspace from the last time they visit. It essentially acts as a mechanism for users to understand what happened in a workspace and how they can collaborative work upon. Could you elaborate a little bit more on why you would turn off logging of activity?

**Comment by gabibrant** [comment]
@akshay-kulkarni-03 @sankalp0o I manage a public workspace and mistakenly added an environment with a customer name. But now I cannot delete the activity and it's really bad that it is showing there publicly. I really need to clean up this activity from the logs and deleting and recreating the full workspace is not an option as there are many public links to the collections.
